ci/terraform/oidc/site.tf

```diff
@@ -44,7 +44,7 @@ locals {
   deploy_account_interventions_count   = 1
   deploy_ticf_cri_count                = contains(["sandpit", "authdev1", "authdev2", "dev", "build", "staging", "integration"], var.environment) ? 1 : 0
   deploy_reauth_user_count             = contains(["build", "sandpit", "authdev1", "authdev2", "staging", "integration"], var.environment) ? 1 : 0
-  deploy_check_email_fraud_block_count = contains(["build", "sandpit", "authdev1", "authdev2", "staging"], var.environment) ? 1 : 0
+  deploy_check_email_fraud_block_count = contains(["build", "sandpit", "authdev1", "authdev2", "staging", "integration", "production"], var.environment) ? 1 : 0
```

Comment:
@pskushwaha1, these are the correct new values right?

Ideally I'd remove all the flags but this got very fiddly with the integration tests, so flag removal will have to follow later.

Pretty much done in https://github.com/govuk-one-login/authentication-api/pull/4999

Reply:
As discussed we can go with` deploy_check_email_fraud_block_count = 1`